src/instrumenter/transpile.ts

```diff
@@ -308,9 +303,9 @@ export function generateExprAST(
 
         let referencedDeclaration = -1;
 
-        if (type instanceof SFunctionSetType) {
+        if (type instanceof FunctionSetType) {
             referencedDeclaration = type.definitions[0].id;
```

Comment:
Maybe it would be better if this was `single(type.definitions).id`. Who is responsible for guaranteeing overloaded functions in the set are filtered here? Or is this done only at the parent level (FunctionCall)?

Reply:
Ensuring that there is a single definition is done in `tcFunctionCall` here: https://github.com/ConsenSys/scribble/blob/f4281ca2b31329f988487e14c4b77e1ab10357d7/src/spec-lang/tc/typecheck.ts#L1629

Reply:
Fixed in ef446bd.